src/main/java/com/softserve/academy/spaced/repetition/utils/validators/EmailNotExist.java

```diff
@@ -9,7 +9,7 @@
 @Documented
 @Constraint(validatedBy = {EmailNotExistValidator.class})
 public @interface EmailNotExist {
-    String message() default "{message.exception.emailNotExists}";
```

Comment:
You need to investigate this Validator
I think it has not correct name because it return true if email not exist and false if email already exist
maybe better will be rename validator to CheckOnExisting or smth like that